my-app/src/routes/groups/[id=integer]/balance/+page.svelte

```diff
@@ -13,21 +14,16 @@
 	export function balanceColor(balance: number) {
 		return balance > 0 ? '#056517' : balance < 0 ? '#bf1029' : null;
 	}
-	export const personalBalanceTooltip =
-		data?.userBalance < 0
-			? 'Les debes dinero'
-			: data?.userBalance > 0
-				? 'Te deben dinero'
-				: 'Están a mano';
+	export const personalBalanceTooltip = balanceTooltip(data.userBalance);
 
 	export function balanceTooltip(balance: number) {
-		return balance > 0 ? 'Le debes dinero' : balance < 0 ? 'Te debe dinero' : 'Están a mano';
+		return balance > 0 ? 'Debes dinero' : balance < 0 ? 'Te debe dinero' : 'Están a mano';
```

Comment:
Habría que actualizar estos textos, porque en realidad se le debe al grupo.

```suggestion
		return balance > 0 ? 'Debe dinero' : balance < 0 ? 'Se le debe dinero' : 'Sin deudas';
```